src/elements/post-card/post-card.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { Hero, Premiere, Item, Quote, Default } from './variants'
+
+type variant = 'hero' | 'premiere' | 'item' | 'card' | 'quote'
```

Comment:
default should be an option here.
```suggestion
type variant = 'hero' | 'premiere' | 'item' | 'card' | 'quote' | 'default'
```